Cargo.toml

```diff
@@ -40,7 +40,7 @@ envsubst = "0.2.0"
 env_logger = "0.9.0"
 fail = { version = "0.5", features = ["failpoints"] }
 fn-error-context = "0.2.0"
-futures = "0.3.18"
+futures = "0.3.17"
```

Comment:
Is this a result of a merge conflict?  I think downgrading should have a strong rationale.

Reply:
Yeah, I just made it build, not trying to propose this as the final push. But I made ostree-ext point to git and called: `cargo update` which started complaining about `0.3.18`. Moving it down to 0.3.17 made it build.